mod_form.php

```diff
@@ -422,7 +422,7 @@ public function definition() {
         // Getting Course participants.
         $courseparticipants = [];
         foreach ($participants as $participant) {
-            $courseparticipants[] = ['participantid' => $participant->id, 'participantemail' => $participant->email];
+            $courseparticipants[] = ['participantid' => $participant->id, 'participantname' => $participant->firstname . ' ' . $participant->lastname];
```

Comment:
It seems that Moodle prefers that you use the fullname() function when you want to show the "firstname lastname". This allows for internationalization.
```suggestion
            $courseparticipants[] = [
                'participantid' => $participant->id,
                'participantname' => fullname($participant),
            ];
```

Alternatively, if the fullname() isn't unique enough, we could use the name-addr email address format:
```suggestion
            $courseparticipants[] = [
                'participantid' => $participant->id,
                'participantname' => fullname($participant) . ' <' . $participant->email . '>',
            ];
```

Reply:
As discussed in our meeting: @armccoy will test the name-addr email address format to make sure it works well in this context.

Reply:
Just tested the settings layout with full name and email. No layout issues. Let's go with that suggestion.